gulpfile.js

```diff
@@ -29,7 +29,7 @@ const dirs = {
 
 const licenseTemplate = `/*!
  *
- * Neutrino v@PACKAGE_VERSION@ | @FILE_NAME@
+ * Carbon-Angular v@PACKAGE_VERSION@ | @FILE_NAME@
```

Comment:
Just to make sure the incense header and package name line up ...

```suggestion
 * carbon-components-angular v@PACKAGE_VERSION@ | @FILE_NAME@
```

Reply:
Actually, why aren't we using full name everywhere to minimize confusion?

Reply:
For the stuff in webpack I think `carbon-angular` is fine - `carbon-components-angular` is kinda long (since it's going to be in the filenames and how people access it. Everywhere else though yeah, it could be the full name